presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/TestSqlServerIntegrationSmokeTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.sqlserver;
+
+import io.prestosql.tests.AbstractTestIntegrationSmokeTest;
+import org.testng.annotations.Test;
+
+import static io.airlift.tpch.TpchTable.ORDERS;
+import static io.prestosql.plugin.sqlserver.SqlServerQueryRunner.createSqlServerQueryRunner;
+import static io.prestosql.plugin.sqlserver.TestingSqlServerServer.executeInSqlServer;
+import static java.lang.String.format;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+@Test
+public class TestSqlServerIntegrationSmokeTest
+        extends AbstractTestIntegrationSmokeTest
+{
+    public TestSqlServerIntegrationSmokeTest()
+    {
+        super(() -> createSqlServerQueryRunner(ORDERS));
+    }
+
+    @Test
+    public void testDropTable()
+    {
+        assertUpdate("CREATE TABLE test_drop AS SELECT 123 x", 1);
+        assertTrue(getQueryRunner().tableExists(getSession(), "test_drop"));
+
+        assertUpdate("DROP TABLE test_drop");
+        assertFalse(getQueryRunner().tableExists(getSession(), "test_drop"));
+    }
+
+    @Test
+    public void testInsert()
+    {
+        executeInSqlServer("CREATE TABLE test_insert (x bigint, y varchar(100))");
```

Comment:
- Why don't you `CREATE TABLE` in Presto?
- Then you should be able to move this to `AbstractTestIntegrationSmokeTest`

Reply:
> Why don't you CREATE TABLE in Presto?

Because test is focused on INSERT.

Reply:
Sure, but you can still CTAS an empty table and then test INSERT via Presto.
With this,  `testInsert` would be independent of the connector under test and could be added to `AbstractTestIntegrationSmokeTest`